package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "fs-merger",
-  "version": "3.1.0",
+  "version": "4.0.0",
```

Comment:
Were any of hte changes breaking changes? I was trying to keep it possible to get this update within the `3.x` series (so folks can transitively update by lock file shenanigans...).

Reply:
@rwjblue I can make it 3.2 instead.